exercises/helloworld-workflow/solution/Workflow/Activities.cs

```diff
@@ -0,0 +1,9 @@
+namespace Temporalio.HelloWorld.Workflow;
```

Comment:
I think this should be something like `TemporalioHelloWorld` or `Temporal101` or similar so that it doesn't pollute the root SDK namespace. It doesn't really matter, but we call our sample's namespace `TemporalioSamples` not `Temporalio.Samples` for this reason.